### PR TITLE
design-doc: Add section on cloud agents

### PR DESCRIPTION
**Pull request by ajeddeloh**

### CODE REVIEW

**Comment by dustymabe** [comment]
LGTM - will leave open for other reviews til tuesday

**Comment by ajeddeloh** [comment]
I'll wait till wednesday actually. People might be gone for the long weekend.

File: Design.md [hunk 1]
```diff
@@ -39,3 +39,19 @@ the root filesystem has changed. This is currently untested.
...
  - What filesystem do we use for / and /var? ext4?
+
+## Cloud Agents
+
+Originally discussed in [#12](https://github.com/coreos/fedora-coreos-tracker/issues/12).
+
+### Summary:
+
+ - FCOS will not ship cloud agents whenever possible.
+ - Some clouds require the OS perform tasks like phoning home. For those we will re-implement that functionality in
```
Comment (cgwalters) [inline Design.md]: s/phoning home/signaling boot completion/ ?

**Comment by cgwalters** [comment]
One optional, LGTM.

**Comment by dustymabe** [comment]
@ajeddeloh - want to merge this ?

**Pull request merged**